Vwitter/View Controllers/ProfileViewController.m

```diff
@@ -0,0 +1,31 @@
+//
+//  ProfileViewController.m
+//  Vwitter
+//
+//  Created by Christina Li on 7/8/22.
+//
+
+#import "ProfileViewController.h"
+
+@interface ProfileViewController ()
+
+@end
+
+@implementation ProfileViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+}
+
+/*
+#pragma mark - Navigation
+
+// In a storyboard-based application, you will often want to do a little preparation before navigation
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
+    // Get the new view controller using [segue destinationViewController].
+    // Pass the selected object to the new view controller.
+}
+*/
```

Comment:
nit: remove

Reply:
This comes auto-generated with the UIViewController .m files; is it convention to remove it even when I might anticipate on using it later?